.ci/magic-modules/welcome-contributor.sh

```diff
@@ -2,14 +2,20 @@
 
 set -x
 
+ASSIGNEE=$(shuf -n 1 <(printf "danawillow\nrambleraptor\nemilymye\nrileykarson\nSirGitsalot\nslevenick\ntysen"))
+
 cat > comment/pr_comment << EOF
 Hello!  I am a robot who works on Magic Modules PRs.
 
 I have detected that you are a community contributor, so your PR will be assigned to someone with a commit-bit on this repo for initial review.  They will authorize it to run through our CI pipeline, which will generate downstream PRs.
 
 Thanks for your contribution!  A human will be with you soon.
-EOF
 
-shuf -n 1 <(printf "danawillow\nrambleraptor\nchrisst\nrileykarson\nSirGitsalot\nslevenick\ntysen") > comment/assignee
+$ASSIGNEE, please review this PR or find an appropriate assignee.
```

Comment:
Does there need to be an @ here for GitHub to recognize it as a user?

Reply:
yes there does, thanks!